### DISCUSSION TITLE
Error installing netcdf-c-4.9.0

### DISCUSSION THREAD

**Discussion by leehaksa12** [Unidata/netcdf-c#2587] [General]
Hello, I am trying to install netcdf-c to run the wrf model.

After installing zlib and hdf5, I proceeded with configure.

However, with these errors, the make check installation does not work.

I'd appreciate it if you could take a look at the problem.
`
/usr/bin/ld: ../liblib/.libs/libnetcdf.so: undefined reference to `H5free_memory'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:765: ncgen3] Error 1
make[1]: Leaving directory '/home/lee/netcdf-c-4.9.0/ncgen3'
make: *** [Makefile:764: check-recursive] Error 1
`

**Comment by DennisHeimbigner** [comment]
Sorry, I meant what version of HDF5 are you using?

**Reply by leehaksa12** [reply]
1.8.12 version in use
